AnnoyancesFilter/Cookies/sections/cookies_specific.txt

```diff
@@ -7979,6 +7979,7 @@ gear.cdprojektred.com##div[class^="main-cookies-"]
 vtb.ru###cookiemsgoffpls
 komoot.de##.css-t7ztk8
 erco.com##.cookie-container_
+||check24.de##.c24-cookie-consent-wrapper
```

Comment:
Element hiding rules do not use ||

Reply:
@BlazDT  Sorry for the mistake; changed it